charts/cluster-api-cluster-openstack/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: cluster-api-cluster-openstack
 description: A Helm chart to deploy a Kubernetes Cluster
 type: application
-version: v0.4.3
+version: v0.4.5
```

Comment:
Also, probably my bad, it's a double version skip, but I'm not terribly bothered!

Reply:
aah yeah I think in part of the diff merging things have skipped a step. I'll drop it back one for sanity sake!

Reply:
dropped it to v0.4.4